cost-analyzer/pod-utilization.json

```diff
@@ -178,14 +178,14 @@
       ],
       "targets": [
         {
-          "expr": "sum(\n  avg_over_time(container_memory_allocation_bytes{namespace=\"$namespace\", pod=\"$pod\", container!=\"POD\"}[$__range])\n) by (container,node)",
+          "expr": "sum(\n  avg_over_time(container_memory_allocation_bytes{namespace=~\"$namespace\", pod=~\"$pod\", container!=\"POD\", cluster_id=~\"$cluster\"}[$__range])\n) by (container,node, cluster_id)",
```

Comment:
`cluster_id` is not always used to identify clusters -- that is a configurable field

Reply:
Thanks for calling that out... @zdenekvicar can we look to make that "cluster_id" configurable? maybe sub in $clusterLabel or the like?

Reply:
After that I'd be supportive of approving and merging.

Reply:
I am all in to change that so it can use whatever label is selected, but I am not sure if I know how :/ so any help is definitely welcome here (maybe I will figure this out later).

I see the label is controlled in here `prometheus.server.global.external_labels` in values file, but I am not really sure how to propagate whatever cluster-label is defined here, to the json dashboard source

Reply:
I think it's ok to allow the user to specify and not necessarily propagate? Does that make this easier?